.github/workflows/release.yml

```
@@ -21,24 +21,18 @@ on:
   workflow_dispatch:
 jobs:
   release:
-    runs-on: macos-11
+    runs-on: macos-12
```

```suggestion
    runs-on: macos-14
```
you can use macos-14 as it's the M1 macs that are faster on building/testing